src/components/ExtraNavigation/ExtraNavigation.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+import { Container, Row, Col } from 'react-grid-system';
+
+import Icon from '../Icons';
+
+import './ExtraNavigation.scss';
+
+const ExtraNavigation: FunctionComponent<ExtraNavigationProps> = ({
+    label,
+    href = '#',
+}: ExtraNavigationProps) => {
+    const className = 'extra-navigation';
+    return (
+        <div className={className}>
+            <Container>
+                <Row>
+                    <Col>
+                        <div className={`${className}__box`}>
+                            <a
+                                href={href}
+                                className={`${className}__back-anchor`}
+                            >
+                                <Icon name="back" size={16} color="#343638" />
+                            </a>
+                            <div className={`${className}__home-box`}>
+                                <a
+                                    href="#"
+                                    className={`${className}__home-anchor`}
+                                >
+                                    <Icon
+                                        name="home"
+                                        size={16}
+                                        color="#343638"
+                                    />
+                                </a>
+                                <span className={`${className}__label`}>
+                                    {label}
+                                </span>
+                            </div>
+                            <a href="#" className={`${className}__filter-box`}>
+                                <Icon name="filter" size={16} color="#343638" />
```

Comment:
Pasikurk `~/constants/Colors.ts` kur uznaudosi exportuotas spalvas is `res/colors.scss`